### ISSUE TITLE
Binary wheels are not portable (i.e. they are broken)

### ISSUE THREAD

**Issue by dhermes** [dhermes/bezier#98]
On another project (which has private CircleCI builds), using the `bezier==0.6.2` wheel to install on the [`dhermes/python-multi`][1] docker image, the unit tests stall and fail.

I am fairly certain this is caused by the `-march=native` flag.

[1]: https://hub.docker.com/r/dhermes/python-multi/

**Comment by dhermes** [comment]
I made a [canary build][1] to hopefully confirm this failure.

[1]: https://circleci.com/gh/dhermes/bezier/1171

**Comment by dhermes** [comment]
The `-march=native` flag was added in 387798248cf452b27b8e7aa16b83417b1cdcb196, which came **after** `0.5.0`:

```
 $ git log -1 --pretty=%H 0.5.0
1b13a00caad761eaeb0da4526085864ced5b7241
$ git merge-base 387798248cf452b27b8e7aa16b83417b1cdcb196 0.5.0
1b13a00caad761eaeb0da4526085864ced5b7241
```

and **before** `0.6.0`:

```
$ git merge-base 387798248cf452b27b8e7aa16b83417b1cdcb196 0.6.0
387798248cf452b27b8e7aa16b83417b1cdcb196
```

**Comment by dhermes** [comment]
[Fixed][1] by the [`0.6.3`][2] release!

[1]: https://circleci.com/gh/dhermes/bezier/1181
[2]: https://pypi.org/project/bezier/0.6.3/
